docs/administrative-requirements.md

```diff
@@ -23,66 +23,61 @@ Preparing to connect your system to Enterprise Event Bus may require additional
+Approval typically takes place in two stages: first for use in non-production environments, then for use in production.
 
-**For new LHDI clients**:
+Production approval will require that proof of ATO has been registered in eMASS.
 
-* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request process and LHDI-specific onboarding. 
+### Sensitivity Filtering
 
-## Authority to Operate (ATO)
+The implementation of Sensitivity Filtering is required when event data, or data spawned from a callback initiated by the data, will be displayed through a VA application user interface to a VA employee or contractor.
 
-ATO, or “Authority to Operate,” indicates that your system has been evaluated by VA and given permission to deploy code in production.
+[Documentation for the rationale behind sensitivity filtering (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/Benefits%20Information%20Sensitivity%20Filtering.md) and [instructions for implementing Sensitivity Filtering (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/Benefits%20Information%20Sensitivity%20Level%20Filtering%20Options.md) can be found in the Enterprise Event Bus GitHub repository. 
 
-Teams that have an active ATO should review it to understand the implications of integrating with other systems.
+## ESECC
 
-**For non-LHDI clients**:
+A request to the Enterprise Security External Change Council (ESECC) may be needed to authorize an opening in the firewall to enable a connection between the client system and the Enterprise Event Bus.
 
-ATO is evaluated once eMASS system registration has been completed and approved. Teams that do not have an active ATO should begin the process as early as possible in the Enterprise Event Bus integration process.
+Prior to determining if your system will require an ESECC, teams are highly encouraged to attempt connecting to the Enterprise Event Bus. If an error message indicates that traffic is blocked at the network level, then check to see if an ESECC is required.
 
-**For new LHDI clients**:
+![A diagram showing various scenarios indicating whether a system would need an ESECC. If you are already on the Lighthouse Delivery Infrastructure (LHDI), no ESECC is required. If you are on AWS GovCloud but outside of LHDI, check to see if an ESECC is required. If you are on Non-AWS Cloud (e.g. Azure), check to see if an ESECC is required. If you are on-premises, e.g., VistA, an ESECC is required.](img/Client-Environments-ESECC-Decision-Circles.svg)
```

Comment:
Has this diagram been verified recently (as in the last 2-3 months)?

Reply:
It was last updated on [April 2](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/commit/c823543a6e841734c775875f1746d977945cc262). If we want to update ESECC I'll create a new ticket to update those sections.

Reply:
I think a new ticket to review those sections, and update if needed, would be a good idea. Our guidance from LHDI on ESECCs has been shifting around over the past few months and this information may still be correct, or it may not.

Reply:
Added [#3044](https://app.zenhub.com/workspaces/event-bus-team-632b50a9a7bf8400277f2cf8/issues/gh/department-of-veterans-affairs/ves/3044) to fix the ESECC section.